spec/requests/api/service_dialogs_spec.rb

```diff
@@ -181,6 +181,71 @@
         expect(response).to have_http_status(:ok)
       end
     end
+
+    context 'Service Dialogs Copy' do
+      it 'forbids blueprint copy without an appropriate role' do
+        dialog = FactoryGirl.create(:dialog_with_tab_and_group_and_field)
+        api_basic_authorize
+
+        run_post(service_dialogs_url(dialog.id), :action => 'copy')
+
+        expect(response).to have_http_status(:forbidden)
+      end
+
+      it 'Can copy multiple service dialogs' do
+        dialog1 = FactoryGirl.create(:dialog_with_tab_and_group_and_field)
+        dialog2 = FactoryGirl.create(:dialog_with_tab_and_group_and_field)
+        api_basic_authorize collection_action_identifier(:service_dialogs, :copy)
+
+        expected = {
+          'results' => a_collection_containing_exactly(
+            a_hash_including(
+              'label' => "Copy of #{dialog1.label}"
```

Comment:
If you're testing against this attribute it might be better to set it explicitly above when you create the dialog. Then, using the literal without interpolation ensures everything's wired up correctly